app.js

```diff
@@ -50,36 +56,56 @@ function showButton() {
     submitButton.classList.add('show-button');
 }
 
-function startPlaying() {
-    
+function getComputerValue() {  
+    switch(true) {
+        case randomNumber === 0:
+            appState.computerSelectedItem = paper;
+            console.log(randomNumber);
+            break;
+        case randomNumber === 1:
+            appState.computerSelectedItem = rock;
+            console.log(randomNumber);
+            break;
+        default:
+            appState.computerSelectedItem = scissors;
+            console.log(randomNumber);
+            break;
+    }
+    console.log(appState);
+}
+
+function getRandom() {
+    min = Math.ceil(0);
+    max = Math.ceil(2);
```

Comment:
you do not need `ceils` for 2 or 0, the are not floating numbers (like: `2.13`)
You can put 0 and 2 directly to Math.Floor. Use your mathematic knowledge to make `Math.floor...` shorter ;)